bok_choy/a11y/helpers.py

```diff
@@ -0,0 +1,10 @@
+"""
+Common helper methods for accessibility audit functionality.
+"""
+
+
+class AccessibilityError(Exception):
```

Comment:
It looks like this file can be removed now as AccessibilityError is defined in a11y/a11y_audit.py.